applications/OptimizationApplication/python_scripts/algorithms/standardized_constraint.py

```diff
@@ -148,13 +157,14 @@ def GetAbsoluteChange(self) -> float:
 
     def GetInfo(self) -> dict:
         info = {
-            "name": self.GetReponse().GetName(),
+            "name": self.GetResponseName(),
             "value": self.GetValue(),
+            "scaled_value": self.GetValue() * abs(self.__scaling),
```

Comment:
```suggestion
            "scaled_value": self.GetValue() * self.__scaling,
```

Since we check whether `self.__scaling` is negative in the `__init__`, and if so, an error is thrown.

Reply:
self.__scaling can be negative to respect low or larger constraints. Here, I want to print scaled value with positive scaling, without our internal logic of negative scaling